src/Microsoft.Framework.Runtime.Roslyn/RoslynCompiler.cs

```diff
@@ -213,18 +213,59 @@ private CompilationModules GetCompileModules(ILibraryKey target)
         private static CSharpCompilation ApplyVersionInfo(CSharpCompilation compilation, Project project,
             CSharpParseOptions parseOptions)
         {
+            const string assemblyFileVersionName = "System.Reflection.AssemblyFileVersionAttribute";
+            const string assemblyVersionName = "System.Reflection.AssemblyVersion";
+            const string assemblyInformationalVersion = "System.Reflection.AssemblyInformationalVersion";
+
             var emptyVersion = new Version(0, 0, 0, 0);
```

Comment:
Also, a more typical empty version is to call the parameterless ctor (so that it has no build/revision in it, as opposed to explicitly 0).